src/components/Discussion/Tree/CommentNode.js

```diff
@@ -0,0 +1,529 @@
+import React, { useMemo } from 'react'
+import { DiscussionContext } from '../DiscussionContext'
+import * as config from '../config'
+import { useColorContext } from '../../Colors/ColorContext'
+import { readDraft } from '../Composer/CommentDraftHelper'
+import scrollIntoView from 'scroll-into-view'
+import { css, merge } from 'glamor'
+import * as Comment from '../Internal/Comment'
+import { CommentComposer } from '../Composer'
+import { CommentList } from './CommentList'
+import { mUp } from '../../../theme/mediaQueries'
+import {
+  EditIcon,
+  EtiquetteIcon,
+  FeaturedIcon,
+  ReportIcon,
+  UnfoldLessIcon,
+  UnfoldMoreIcon,
+  UnpublishIcon
+} from '../../Icons'
+import IconButton from '../../IconButton'
+import { timeFormat } from '../../../lib/timeFormat'
+
+const dateFormat = timeFormat('%d.%m.%Y')
+const hmFormat = timeFormat('%H:%M')
+
+const buttonStyle = {
+  display: 'block',
+  background: 'transparent',
+  border: 'none',
+  cursor: 'pointer',
+  outline: 'none',
+  padding: 0
+}
+
+const styles = {
+  highlightContainer: css({
+    padding: '7px 7px 0 7px',
+    margin: '0 -7px 12px -7px'
+  }),
+  boardColumn: css({
+    [mUp]: {
+      flex: '1 0 auto',
+      padding: 10,
+      width: '50%'
+    }
+  }),
+  showMUp: css({
+    display: 'none',
+    [mUp]: {
+      display: 'block'
+    }
+  }),
+  hideMUp: css({
+    [mUp]: {
+      display: 'none'
+    }
+  }),
+  modalRoot: css({
+    marginBottom: 15
+  }),
+  hiddenToggle: css({ display: 'none' }),
+  commentWrapper: ({ isExpanded }) =>
+    css({
+      /*
+       * On larger screens, hide the action button and reveal only on hover.
+       */
+      [mUp]: isExpanded && {
+        '@media (hover)': {
+          [`& [data-${Comment.headerActionStyle({ isExpanded })}]`]: {
+            display: 'none'
+          },
+          [`&:hover [data-${Comment.headerActionStyle({ isExpanded })}]`]: {
+            display: 'block'
+          }
+        }
+      }
+    }),
+  root: ({ isExpanded, nestLimitExceeded, depth, board }) =>
+    css({
+      background: 'transparent',
+      position: 'relative',
+      margin: `10px 0 ${(isExpanded ? 24 : 16) + (depth === 0 ? 20 : 0)}px`,
+      paddingLeft: nestLimitExceeded || depth < 1 ? 0 : config.indentSizeS,
+      [mUp]: {
+        paddingLeft: nestLimitExceeded || depth < 1 ? 0 : config.indentSizeM,
+        ...(board
+          ? {
+              display: 'flex',
+              marginLeft: -10,
+              marginRight: -10,
+              marginBottom: 50
+            }
+          : {})
+      }
+    }),
+  verticalToggle: ({ drawLineEnd, depth, isExpanded, isLast }) =>
```

Comment:
icon size should be increased to 20px instead of 1em . needs adjusting of `left` 
before
<img width="77" alt="Screenshot 2021-11-19 at 11 14 52" src="https://user-images.githubusercontent.com/20746301/142605928-137df5bf-d392-4a82-9a0b-f37fc8082f9c.png">


after
<img width="102" alt="Screenshot 2021-11-19 at 11 14 01" src="https://user-images.githubusercontent.com/20746301/142605891-c8d48440-941d-4f97-8184-f8ff61acea2a.png">